### ISSUE TITLE
Code question

### ISSUE THREAD

**Issue by Amos1109** [LTH14/targeted-supcon#5]
Hi, thanks for your share,and I find some codes in moco/builder.py, from L417-L419.     

mask_pos_view = torch.cat([torch.ones([mask_pos_view.shape[0], 1]).cuda(), mask_pos_view], dim=1)
mask_pos_view_class = torch.cat([torch.ones([mask_pos_view_class.shape[0], 1]).cuda(), mask_pos_view_class], dim=1)
mask_pos_view_target = torch.cat([torch.zeros([mask_pos_view_target.shape[0], 1]).cuda(), mask_pos_view_target], dim=1)

Why we need these operations?
